docs/source/workflow_inputs.rst

```diff
@@ -433,13 +433,47 @@ Site information is entered in ``/HPXML/Building/BuildingDetails/BuildingSummary
   ================================  ========  ===========  ===========  ========  ========  ============================================================
   ``SiteType``                      string                 See [#]_     No        suburban  Terrain type for infiltration model
   ``ShieldingofHome``               string                 See [#]_     No        normal    Presence of nearby buildings, trees, obstructions for infiltration model
-  ``extension/GroundConductivity``  double    Btu/hr-ft-F  > 0          No        1.0       Thermal conductivity of the ground soil [#]_
+  ``Soil``                          element                0 - 1        No                  Soil properties
   ``extension/Neighbors``           element                >= 0         No        <none>    Presence of neighboring buildings for solar shading
   ================================  ========  ===========  ===========  ========  ========  ============================================================
 
   .. [#] SiteType choices are "rural", "suburban", or "urban".
   .. [#] ShieldingofHome choices are "normal", "exposed", or "well-shielded".
-  .. [#] GroundConductivity used for foundation heat transfer and ground source heat pumps.
+  
+Soil information is entered in ``Soil``.
+
+  ==================================================================  ================  ===========  ===============  ========  ========  ============================================================
+  Element                                                             Type              Units        Constraints      Required  Default   Notes
+  ==================================================================  ================  ===========  ===============  ========  ========  ============================================================
+  ``Conductivity`` or ``SoilType``/``MoistureType``                   string or double  Btu/hr-ft-F  See [#]_ or > 0  No        unknown   Themal conductivity [#]_ or soil/moisture type
+  ``extension/Diffusivity`` or ``SoilType``/``MoistureType``          string or double  ft2/hr       See or > 0       No        mixed     Diffusivity [#]_ or soil/moisture type
+  ==================================================================  ================  ===========  ===============  ========  ========  ============================================================
+
+  .. [#] | SoilType choices are "sand", "silt", "clay", "loam", "gravel", or "unknown".
+         | MoistureType choices are "dry", "wet", or "mixed".
+  .. [#] Conductivity used for foundation heat transfer and ground source heat pumps.
+  .. [#] Diffusivity used for ground source heat pumps.
```

Comment:
Seems like it should be used for foundation heat transfer too. Kiva has inputs for [Soil Density and Soil Specific Heat](https://github.com/NREL/OpenStudio-HPXML/blob/319f860c55f4bc56a21a1c119a6c365a29169f2d/HPXMLtoOpenStudio/resources/constructions.rb#L1466-L1467).

Reply:
So we want to pass `ground_diffisivity` into `soil_mat = BaseMaterial.Soil(soil_k_in)`, and make changes to `rho` and `cp` based on "Thermal diffusivity is conductivity / (density * specific heat)"? Something like that?

Reply:
Yep, that's right. I don't know what the best approach is since there are two unknowns (density and specific heat), but right now we essentially use a different thermal diffusivity in Kiva than we do for GSHPs.

Reply:
Does it make sense to fix either `rho` or `cp`, and then calculate either `cp` or `rho`, respectively, based on the value passed in for `ground_diffusivity`?

Reply:
@joseph-robertson: I like that approach. I don't think it matters all that much which one we fix since conductivity has a much larger impact than either ```rho``` or ```cp```. That said, I think we should fix ```cp``` and let ```rho``` vary, since we weren't even explicitly providing ```rho``` to the GSHP model before this project started (but you could calculate it based on diffusivity and ```cp```). Maybe worth a very quick test to confirm though: if we ran a case with a low and high fixed ```rho```, how much does the annual energy consumption change? My sense is we're talking ~1% impact here.

Reply:
Based on `soil_mat` properties `k`, `rho`, `cp` assigned to KivaSettings, a diffusivity value of 0.0869 is calculated. The default diffusivity value for GSHP is 0.0208. If we were to re-calculate `rho` (by fixing `cp) on KivaSettings, it would be 480 (vs. 1842). So, our options are:

a) implement this change in `rho` on KivaSettings,
b) change the diffusivity default for GSHP to 0.0869, or
c) leave everything as it is (i.e., don't update KivaSettings based on GSHP diffusivity)

Reply:
I tried option "A" to see what the impact would be on the building loads, since I don't have a great sense of how impactful density is on Kiva. Turns out it matters quite a bit! Just changing the density from our default value (115 lbm/ft3 or 1842 kg/m3) to the value you'd calculate based on the default diffusivity, specific heat, and conductivity (30 lbm/ft3 or 480 kg/m3) there's a big difference in the building loads. Using base.xml (which has a slab foundation), the density change leads to a >20% change in cooling energy consumption. 

I think this rules out "A", so my vote is for option "C". I also don't like that with option A the density comes out to be substantially less dense than water, which doesn't seem very likely regardless of soil type.

Reply:
What if we fixed `rho` and let `cp` vary?

Reply:
@joseph-robertson: Same problem, you get a >20% change in the cooling energy consumption (although a decrease rather than an increase). So no matter which one you fix this seems to lead to pretty substantial differences if we try to harmonize this, leading me to think option "C" is still the better option here.